web/src/actions/index.js

```diff
@@ -0,0 +1,6 @@
+export const selectBase = (base) => {
+  return {
+    type: 'BASE_SELECTED',
```

Comment:
should be making BASE_SELECTED a constant, that you import into actions and the reducer